tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/klass/ClassTest.java

```diff
@@ -64,6 +64,16 @@ public void testTypeRef() {
         BRunUtil.invoke(compileResult, "testTypeRefInClass");
     }
 
+    @Test
+    public void testModuleVariableReferencingClass() {
+        BRunUtil.invoke(compileResult, "testModuleVariableReferencingClass");
+    }
+
+    @Test
```

Comment:
Is this the correct file for this test? Maybe both can go in `org.ballerinalang.test.object.ObjectTest`?

Reply:
Moved to `ObjectTest`